components/country/AppsStatsCircumventionRow.js

```diff
@@ -149,7 +149,7 @@ class AppsStatsCircumventionRow extends React.Component {
               <Box ml='auto'>
                 <FormattedMessage id='Country.Apps.Label.LastTested' />
                 {' '}
-                <strong>{moment(data.last_tested).fromNow()}</strong>
+                <strong>{moment.utc(data.last_tested).fromNow()}</strong>
```

Comment:
Maybe we want to add after here the string UTC to clarify the timezone.

Reply:
Makes sense. I will update similarly in other places too

Reply:
Done!

Reply:
Ah sorry I think I did not explain myself properly. What I meant is that you should add after
```
<strong>{moment.utc(data.last_tested).fromNow()}</strong>
```

Literally the string `UTC` so that the user understands that the time is being displayed in the UTC timezone.

However I see now that actually you are using `fromNow()` which according to the docs says: https://momentjs.com/docs/#/displaying/fromnow/
```
moment([2007, 0, 29]).fromNow();     // 4 years ago
```

So it doesn't actually make sense to do that here.

I think what you want to be doing is actually revert the commit https://github.com/ooni/explorer/pull/373/commits/77a3fb735dfadd61a5acbf48e1016fc0628e9e2d.

We should however verify if the delta of time is correct, but I believe it should be, because by doing `moment().utc(last_tested)` we are telling moment that the timezone of the source string is UTC and it should calculate the delta in accordance to that.

Sorry for the confusion.

Reply:
Ooooh. Okay. Got it.

Reply:
So what I will do is add the string UTC in places where it makes sense. As in where it's actually a date and not delta.

Reply:
@sarathms @hellais I think I should approach it like https://github.com/ooni/explorer/pull/328/commits/e0ee6147a530ec1f09c95807d88c74ffc2ad55b8 instead of just adding a text? Because then it will be similar.